src/solver/conv_hip_implicit_gemm_grouped_fwd_xdlops.cpp

```diff
@@ -189,23 +193,142 @@ bool ConvHipImplicitGemmGroupFwdXdlops::CheckCKApplicability(
 }
 #endif
 
+std::vector<std::string> GetKernelAsTokens(const std::string& kernel)
```

Comment:
```suggestion
static std::vector<std::string> GetKernelAsTokens(const std::string& kernel)
```
Or you can use unnamed namespace here